libusb/os/darwin_usb.c

```diff
@@ -2262,9 +2294,139 @@ static int darwin_free_streams (struct libusb_device_handle *dev_handle, unsigne
 }
 #endif
 
+#if InterfaceVersion >= 700
+
+/* macOS APIs for getting entitlement values */
+
+#if TARGET_OS_OSX
```

Comment:
When could this be false?

Reply:
Building on iOS, tvOS, etc.

Reply:
Does anyone build libusb on these platforms? We haven't directly supported running on either OS and I don't know if IOKit will allow libusb the necessary hardware access.

Reply:
It's not officially supported no, but people have been using it in jailbroken apps. This ensures those configurations won't be broken while at the same time not explicitly introducing iOS specific code.

Reply:
Makes sense. Will continue to look at the rest of the code.